packages/api/lib/testUtils.js

```diff
@@ -1,12 +1,108 @@
 'use strict';
 
+const cloneDeep = require('lodash.clonedeep');
 const { randomString } = require('@cumulus/common/test-utils');
+const { aws: { lambda } } = require('@cumulus/common');
+const { User } = require('../models');
+
+/**
+ * Add a user that can be authenticated against
+ *
+ * @param {Object} params - params
+ * @param {User} params.userDbClient - an instance of the Users model
+ * @returns {Promise<Object>} - an object containing a userName and a password
+ */
+async function createFakeUser({ userDbClient }) {
+  // Create the user and token for this request
+  const userName = randomString();
+  const password = randomString();
+
+  await userDbClient.create([
+    {
+      userName,
+      password,
+      expires: Date.now() + (60 * 60 * 1000) // Token expires in 1 hour
+    }
+  ]);
+
+  return { userName, password };
+}
+
+/**
+ * Call the Cumulus API by invoking the Lambda function that backs the API
+ * Gateway endpoint.
+ *
+ * Intended for use with integration tests.  Will invoke the function in AWS
+ * Lambda.  This function will handle authorization of the request.
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @param {string} params.functionName - the name of the Lambda function that
+ *   backs the API Gateway endpoint.  Does not include the stack prefix in the
+ *   name.
+ * @param {string} params.payload - the payload to send to the Lambda function.
+ *   See https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @returns {Promise<Object>} - the parsed payload of the response.  See
+ *   https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-output-format
+ */
+async function callCumulusApi({ prefix, functionName, payload: userPayload }) {
```

Comment:
Is this something we would use with the unit tests in addition to the integration tests? Wondering if this is the right place for these functions.

Reply:
No, we wouldn't use this with the unit tests, since it calls out to AWS.  Unit tests would probably use the `testEndpoint` function defined below.

Reply:
It seems like this might not be the best place for this. Maybe this should go in the integration test package

Reply:
Moved to `example/spec/helpers/apiTestUtils.js`